docs/aggregation.rst

```diff
@@ -88,16 +90,26 @@ Amundsen Sea Deep, Eastern Ross Sea Shelf, Eastern Ross Sea Deep,
 Western Ross Sea Shelf, Western Ross Sea Deep, East Antarctic Seas Shelf,
 East Antarctic Seas Deep.
 
-Greenland Regions
+ISMIP6 Greenland Regions
 ~~~~~~~~~~~~~~~~~
 
-The function :py:func:`geometric_features.aggregation.ocean.greenland`
+The function :py:func:`geometric_features.aggregation.ocean.ismip6_greenland`
 aggregates regions of the continental shelves around Greenland based on the
 ISMIP6 regions: ISMIP6 Greenland Central East Shelf,
 ISMIP6 Greenland Central West Shelf, ISMIP6 Greenland North East Shelf,
 ISMIP6 Greenland North Shelf, ISMIP6 Greenland North West Shelf,
 ISMIP6 Greenland South East Shelf, ISMIP6 Greenland South West Shelf.
 
+NASA Greenland Regions
```

Comment:
Can you update the descriptions for "NASA Greenland Regions" and "ISMIP6 Greenland Regions" to indicate if each is intended for use by ocean or ice sheet applications or both?